open-banking-accelerator/accelerators/ob-apim/repository/resources/apis/Accounts/accounts-dynamic-endpoint-insequence.xml

```diff
@@ -0,0 +1,35 @@
+<!--
+ ~ Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ ~
+ ~ WSO2 LLC. licenses this file to you under the Apache License,
+ ~ Version 2.0 (the "License"); you may not use this file except
+ ~ in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing,
+ ~ software distributed under the License is distributed on an
+ ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~ KIND, either express or implied. See the License for the
+ ~ specific language governing permissions and limitations
+ ~ under the License.
+ -->
+
+<sequence xmlns="http://ws.apache.org/ns/synapse" name="accounts-dynamic-endpoint-insequence">
+    <property name="endpointURI" expression="get-property('To')"/>
+    
+    <!--Appending Basic Auth for Consent Management APIs -->
+    <property name="Authorization" scope="transport" action="remove"/>
+    <header name="Authorization" value="Basic YWRtaW5Ad3NvMi5jb206d3NvMjEyMw==" scope="transport"/>
```

Comment:
Can we remove this authorization value.

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/80d467bc167898197adc085a069aa2f9091f16ba